### PR TITLE
Alpha kmanstrings

### PR DESCRIPTION
**Pull request by Wagnerp**
* Strings + #1091

image

### CODE REVIEW

File: Source/Krypton Components/Krypton.Toolkit/Controls Toolkit/KryptonManager.cs [hunk 1]
```diff
@@ -10,6 +10,8 @@
  */
 #endregion
 
+using System;
```
Comment (Smurf-IV) [inline Source/Krypton Components/Krypton.Toolkit/Controls Toolkit/KryptonManager.cs:13]: Why is this here ?

**Review by Smurf-IV — CHANGES_REQUESTED** [review]

File: Source/Krypton Components/Krypton.Toolkit/Controls Toolkit/KryptonManager.cs (hunk 1 above)
Comment (Wagnerp) [reply Source/Krypton Components/Krypton.Toolkit/Controls Toolkit/KryptonManager.cs]: Think it has been added by ReSharper, removed now

File: Source/Krypton Components/Krypton.Toolkit/Controls Visuals/KryptonAboutToolkitForm.cs [hunk 2]
```diff
@@ -0,0 +1,20 @@
+﻿using System;
```
Comment (Smurf-IV) [inline Source/Krypton Components/Krypton.Toolkit/Controls Visuals/KryptonAboutToolkitForm.cs]: These should not be needed

**Review by Smurf-IV — CHANGES_REQUESTED** [review]

File: Source/Krypton Components/Krypton.Toolkit/Controls Visuals/KryptonAboutToolkitForm.cs (hunk 2 above)
Comment (Wagnerp) [reply Source/Krypton Components/Krypton.Toolkit/Controls Visuals/KryptonAboutToolkitForm.cs]: Not sure why they are showing up here, as they were all removed in the latest commit?

Comment (Smurf-IV) [reply Source/Krypton Components/Krypton.Toolkit/Controls Visuals/KryptonAboutToolkitForm.cs]: Just going through each change in order ....
